pydicom/fileset.py

```diff
@@ -1486,16 +1486,16 @@ def match(ds, **kwargs):
 
     def find_values(
         self,
-        element: Union[str, int],
+        elements: Union[str, int, List[Union[str, int]]],
```

Comment:
Unsure about intuitive argument naming, as can be either single item or list of items

Reply:
`elements` is fine